src/server/graphql/location/location-type-defs.ts

```diff
@@ -0,0 +1,27 @@
+import { gql } from 'apollo-server';
+
+export const locationTypeDefs = gql`
```

Comment:
Help me understand why you chose to use this definition instead of `schema.graphql` file

Reply:
To be honest, I chose this way because it was simpler to import/export and, at least for this project, I couldn't notice disadvantages over `.graphql` files